src/service/globals/mod.rs

```diff
@@ -198,6 +198,9 @@ impl Service {
 
 	pub fn emergency_password(&self) -> &Option<String> { &self.config.emergency_password }
 
+	#[cfg(any(target_os = "android", target_os = "fuchsia", target_os = "linux"))]
+	pub fn url_preview_bound_interface(&self) -> &Option<String> { &self.config.url_preview_bound_interface }
```

Comment:
I would just delete this and access `url_preview_bound_interface` directly through `config`. These unnecessary 2nd-step accessors are slowly being phased out in favour of just accessing the config option directly or if the config option is more complex.

Reply:
I was wondering what these were fore anyway 😅